synapse/lib/syntax.py

```diff
@@ -458,13 +479,18 @@ def parse_oarg(text, off=0):
     '''
     _,off = nom(text,off,whites)
 
-    if is_literal(text,off):
-        valu,off = parse_literal(text,off)
+    if nextchar(text,off,'"'):
+        valu,off = parse_string(text,off)
+        _,off = nom(text,off,whites)
 
     else:
-        valu,off = nom(text,off,varset)
+        valu,off = meh(text,off,'=,)')
+        valu = valu.strip()
+        try:
+            valu = int(valu,0)
+        except ValueError as e:
+            pass
```

Comment:
I would normally never do this, but it's by *far* the fastest way to "make an int if the string is in valid int form".  ( ie, "10", "0xf0f0", "0b10101", etc etc )